assignments/function-conversion.js

```diff
@@ -27,4 +27,11 @@
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+
+exampleArray = [1,2,3,4];
+const triple = exampleArray.map(num) => {
```

Comment:
Look at other arrow syntax examples that work with .map and see where your error is (hint: it's in how this is wrapped)